packages/monaco-graphql/package.json

```diff
@@ -16,6 +16,41 @@
       "url": "https://github.com/acao"
     }
   ],
+  "exports": {
+    ".": {
+      "import": "./esm/monaco.contribution.js",
+      "require": "./dist/monaco.contribution.js",
+      "types": "./esm/monaco.contribution.d.ts"
+    },
+    "./*": {
+      "default": "./*",
+      "import": "./*",
+      "require": "./*",
+      "types": "./*"
+    },
+    "./esm/graphql.worker": {
+      "import": "./esm/graphql.worker.js",
+      "require": "./dist/graphql.worker.js",
+      "types": "./esm/graphql.worker.d.ts"
+    },
+    "./lite": {
+      "import": "./esm/lite.js",
+      "types": "./esm/lite.d.ts",
+      "require": "./dist/lite.js"
+    },
+    "./graphql.worker": {
```

Comment:
I'm still getting the following error.

```
Module not found: Error: Default condition should be last one
```

After some research, it turns out the solution is to move this property last in the object.

https://github.com/mpociot/chatgpt-vscode/issues/1#issuecomment-1337384976

Doing so removed the error.

Reply:
that is... so weird 😆

Reply:
@bboure are you using `vsce package`  as your bundler?

Reply:
I'm using webpack.

Reply:
what version? and which node version? because this is working with webpack 5 in next.js on node 20

Reply:
node 18.12.0
webpack 5.80.0

Reply:
It might be an issue specific to my project and/or configuration. It's an electron app

It seems related to `moduleResolution` but no matter how I change it, I'm having issues in one way or another.

Importing `import { initializeMode } from 'monaco-graphql/esm/initializeMode';` works fine in all cases.

Reply:
ah ok, that makes sense, we are still using moduleResolution: node and not setting type: module yet, so this probably confuses some bundler and/or node versions

Reply:
to note, next.js uses their own swc loader, whereas you may be using ts loader, and each of these will interpret exports config differently, so i will add default at the end if vite and next are happy with that as well. we should re-add monaco examples to the netlify build to confirm this works in node 16 as well

Reply:
FWIW, I confirm I'm using ts-loader